src/plot/src/main/python/plot.py

```diff
@@ -10,9 +10,13 @@
 from pyspark import SparkContext
 from pyspark import sql
 from pyspark.sql import DataFrame
-
+import warnings
 import numpy as np
-from sklearn.metrics import confusion_matrix, roc_curve
```

Comment:
Standard trick is to not import these at the module level but rather inside each function. This way, `from foo import *` will work but calling `foo.bar` will fail.

Reply:
Done